source/_integrations/bang_olufsen.markdown

```diff
@@ -85,7 +85,7 @@ And more advanced app-centric features such as:
 - Adjusting specific sound settings
 - Pairing remotes
 
-These features are not available through the API. Some may become available at a later point, but until then the [Bang & Olufsen App](https://www.bang-olufsen.com/en/dk/story/apps) can be used to configure these settings and features.
+Are not available through the API. Some may become available at a later point, but until then the [Bang & Olufsen App](https://www.bang-olufsen.com/en/dk/story/apps) can be used to configure these settings and features.
```

Comment:
How about moving it a little bit upwards and say "Currently, some features of the Mozart platform are not available such as: "

I'm also removing the "through the API" part in my suggestion since you probably refer to not available within Home Assistant?